File: public/aramex/LocationServicesAPI/Location Services API/location-spi-client-csharp/LocationAPIClientCSharp/frmMain.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace LocationAPIClientCSharp
{
    public partial class frmMain : Form
    {
        public frmMain()
        {
            InitializeComponent();
        }

        private void btnValidateAddress_Click(object sender, EventArgs e)
        {
            using (frmValidateAddress _frmValidateAddress = new frmValidateAddress())
            {
                _frmValidateAddress.ShowDialog(this);
            }
        }

        private void btnFetchCountries_Click(object sender, EventArgs e)
        {
            using (frmFetchCountries _frmFetchCountries = new frmFetchCountries())
            {
                _frmFetchCountries.ShowDialog(this);
            }
        }

        private void btnFetchCountry_Click(object sender, EventArgs e)
        {
            using (frmFetchCountry _frmFetchCountry = new frmFetchCountry())
            {
                _frmFetchCountry.ShowDialog(this);
            }
        }

        private void btnFetchCities_Click(object sender, EventArgs e)
        {
            using (frmFetchCities _frmFetchCities = new frmFetchCities())
            {
                _frmFetchCities.ShowDialog(this);
            }
        }

        private void btnFetchOffices_Click(object sender, EventArgs e)
        {
            using (frmFetchOffices _frmFetchOffices = new frmFetchOffices())
            {
                _frmFetchOffices.ShowDialog(this);
            }
        }

        private void btnExit_Click(object sender, EventArgs e)
        {
            this.Close();
        }
    }
}

File: public/aramex/LocationServicesAPI/Location Services API/location-api-client/LocationAPIClient/frmMain.vb
Public Class frmMain

#Region "Methods"
    Private Sub btnValidateAddress_Click(sender As Object, e As EventArgs) Handles btnValidateAddress.Click
        Using _frmValidateAddress As New frmValidateAddress
            _frmValidateAddress.ShowDialog(Me)
        End Using
    End Sub

    Private Sub btnFetchCountries_Click(sender As Object, e As EventArgs) Handles btnFetchCountries.Click
        Using _frmFetchCountries As New frmFetchCountries
            _frmFetchCountries.ShowDialog(Me)
        End Using
    End Sub

    Private Sub btnFetchCountry_Click(sender As Object, e As EventArgs) Handles btnFetchCountry.Click
        Using _frmFetchCountry As New frmFetchCountry
            _frmFetchCountry.ShowDialog(Me)
        End Using
    End Sub

    Private Sub btnFetchCities_Click(sender As Object, e As EventArgs) Handles btnFetchCities.Click
        Using _frmFetchCities As New frmFetchCities
            _frmFetchCities.ShowDialog(Me)
        End Using
    End Sub

    Private Sub btnFetchOffices_Click(sender As Object, e As EventArgs) Handles btnFetchOffices.Click
        Using _frmFetchOffices As New frmFetchOffices
            _frmFetchOffices.ShowDialog(Me)
        End Using
    End Sub

    Private Sub btnExit_Click(sender As Object, e As EventArgs) Handles btnExit.Click
        Me.Close()
    End Sub
#End Region
    
End Class